package.json

```diff
@@ -53,7 +51,9 @@
     "buildScss": "node-sass ./src/sass/cookie-policy.scss --include-path node_modules --output-style compressed --output build/css",
     "buildJs": "rollup -c",
     "build": "npm run clean && npm run test && npm run buildScss && npm run buildJs",
+    "postpublish": "git push origin --all; git push origin --tags",
```

Comment:
Would this just push all my branches and tags on my origin?

Reply:
I feel like this is very specific and ties the package to a certain environment:
- the package knows about being tracked in `git` as a VCS.
- the `origin` remote is baked in. If origin differs for developers this could  lead to confusion

Reply:
I should have noted this approach is from https://blog.npmjs.org/post/184553141742/easy-automatic-npm-publishes

Reply:
I still think this does not fit for the reasons mentioned above + our forking model.

Reply:
Ok, remove the `postpublish` step.